Formula/rtags.rb

```diff
@@ -28,6 +28,45 @@ def install
     end
   end
 
+  def caveats; <<-EOS.undent
+    The instance of rtags that run via brew services listens to
+    the socket file at #{var}/.rdm
```

Comment:
Hidden directories are preferred in `HOME` by users, but a lot less common in places like `var`. `#{var}/rdm` sounds 👍 to me.